plugins/module_utils/schema.py

```diff
@@ -112,6 +112,21 @@ def set_template_anp_epg(self, epg, fail_module=True):
             self.mso.fail_json(msg=msg)
         self.schema_objects["template_anp_epg"] = match
 
+    def set_template_anp_epg_useg_attr(self, usge_attr, fail_module=True):
+        """
+        Get template endpoint group item that matches the name of an EPG uSeg Attribute.
+        :param usge_attr: Name of the EPG uSeg Attribute to match. -> Str
+        :param fail_module: When match is not found fail the ansible module. -> Bool
+        :return: Template EPG uSeg Attribute item. -> Item(Int, Dict) | None
+        """
+        self.validate_schema_objects_present(["template_anp_epg"])
+        kv_list = [KVPair("name", usge_attr)]
+        match, existing = self.get_object_from_list(self.schema_objects["template_anp_epg"].details.get("uSegAttrs"), kv_list)
+        if not match and fail_module:
+            msg = "Provided uSeg Attribute '{0}' not matching existing uSeg Attribute(s): {1}".format(usge_attr, ", ".join(existing))
```

Comment:
"Provided uSeg Attribute '{0}' does not match the existing uSeg Attribute(s): {1}"